### ISSUE TITLE
[1.8.18] iPhone 6 / iOS 9.3.1 - Song list display after refresh - Without Internet, Search and refresh song from Find Music

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#412]
Frequency: Every time

Action:
1. Turn Off Internet
2. Launch Lomotif
3. Click on Start
4. Select few images from Camera Roll
5. Click on Apply button at Top Right corner
6. Go to Find Music
7. No song appear on the screen and Message will appear "The Internet Connection appears to be offline
8. Tap on Search Music bar
9. Click on Cancel button
10. Check whether the song list appear or not
11. Select any song from the list and apply
12. Check whether the created lomotif has selected song or not

Expected result: Without Internet, The song list should not be appear and if available in case then song should not be added

Actual result: Without Internet, Song list available and also user can add song in lomotif

![img_2315](https://cloud.githubusercontent.com/assets/9691390/15316022/7da5819c-1c38-11e6-8d6a-00e3d339ef15.PNG)

![img_2316](https://cloud.githubusercontent.com/assets/9691390/15316020/7d96c486-1c38-11e6-9dad-684e2305ce40.PNG)

![img_2317](https://cloud.githubusercontent.com/assets/9691390/15316023/7dbc1f7e-1c38-11e6-87a8-f021a0975fbe.PNG)

![img_2318](https://cloud.githubusercontent.com/assets/9691390/15316024/7dcccdba-1c38-11e6-956f-5e153eda8178.PNG)

![img_2319](https://cloud.githubusercontent.com/assets/9691390/15316021/7d9d332a-1c38-11e6-98dd-6284c768f8eb.PNG)

**Comment by vysohanych** [comment]
I can't reproduce it on build 1.8.18 on iPad mini iOS 8.3

**Comment by xiuie** [comment]
I can reproduce this on my iPhone 6 / 1.9.2 

The live build (with users) does not have this issue as I am unable to proceed without internet connection. (error message pops up) and blocks me from proceeding.

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa Yes, Agree @xiuie. The problem persists with the testing build only. There is no problem with live build.

**Comment by Dipamit** [comment]
I am not able to reproduce this issue on iPhone 5/8.3 and iPhone 5/9.3.1 for build [1.8.18]

Note: List of songs displayed but can not select, Pop up displayed for error message.

**Comment by Dipamit** [comment]
Not able to reproduce this issue on iPhone 5/8.3 and iPhone 5/9.3.1 for build [1.8.19]

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa I can reproduce this issue for build [1.8.19] on iPhone 6 / iOS 9.3.2

**Comment by tsukisa** [comment]
@hiteshbheda this is known issue when you have previously cached a song list (download top songs list at least once), will not fix this at the moment

**Comment by hiteshbheda** [comment]
@tsukisa Ok. You can close this issue.
